src/main/kotlin/com/atlassian/performance/tools/report/api/judge/RelativePerformanceStabilityJudge.kt

```diff
@@ -42,4 +43,8 @@ class RelativePerformanceStabilityJudge {
             }
         )
     }
+
+    private fun extractDurationBasedOnLabel(dispersions: Map<String, Duration>, label: String,
+                                            cohort: String) =
+        dispersions[label] ?: error("No action $label for the cohort : $cohort")
```

Comment:
When the method signature is longer than it's implementation, it might be a good idea to inline it ;)

Reply:
When this is still considered as a readable then I will fix that :)
```
val regression =
          (experimentDispersions[label]
                ?: error("No action $label for the cohort ${experimentStats.cohort}")) -
                        (baselineDispersions[label] ?: error("No action $label for the cohort ${baselineStats.cohort}"))

```

Reply:
Fair enough. But let's shorten the method name. And let's extract variables so that we can see clearly that `regression = experimentDispersion - baselineDispersion`